Library/Homebrew/formula_installer.rb

```diff
@@ -427,7 +427,7 @@ def install_dependency(dep, inherited_options)
     end
 
     fi = DependencyInstaller.new(df)
-    fi.options           |= tab.used_options
+    fi.options           |= tab.used_options unless ARGV.ignore_tabs?
```

Comment:
Can probably/may need to add this to a few places where `tab.used_options` are used.